authentication.py

```diff
@@ -55,7 +55,7 @@ def request_macaroon():
     response = requests.request(
         url='https://dashboard.snapcraft.io/dev/api/acl/',
         method='POST',
-        json={'permissions': ['package_access']},
+        json={'permissions': ['package_access', 'package_upload']},
```

Comment:
should we add this list as a param of request_macaroon at some point in the future?

Reply:
yes I think so but for now, as we said we can keep it like this and find some good practice on how to use permissions with macaroons...
Let's not mix everything in this PR. I created an issue for it: https://github.com/canonical-websites/snapcraft.io/issues/220

Reply:
xox